presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTableStatistics.java

```diff
@@ -734,7 +735,7 @@ public void testAnalyzeForTablePartitionedByBigint()
 
         // analyze for all partitions
 
-        assertThat(query("ANALYZE " + tableNameInDatabase)).containsExactly(row(15));
+        assertThat(query(format("ANALYZE %s", tableNameInDatabase))).containsExactly(row(15));
```

Comment:
undo. Our convention is to not to use `format` where it can be replaced with simple concatenation like `"message: " + variable`